universal-login-sdk/lib/core/services/GasModeService.ts

```diff
@@ -23,8 +23,8 @@ export class GasModeService {
     const tokensPrices = await this.priceObserver.getCurrentPrices();
 
     return [
-      await this.getMode('cheap', gasPrices.cheap, tokensPrices),
-      await this.getMode('fast', gasPrices.fast, tokensPrices)
+      this.createMode('cheap', gasPrices.cheap, tokensPrices),
+      this.createMode('fast', gasPrices.fast, tokensPrices)
```

Comment:
I am not sure if I need rename getModes => createModes too. I guess no.

Reply:
createMode looks good to me